File: python/n292_NimGame.py
# https://leetcode-cn.com/problems/nim-game/submissions/

class Solution:
    def canWinNim(self, n: int) -> bool:
        return n % 4 != 0
        

if __name__ == "__main__":
    solu = Solution()
    print(solu.canWinNim(26))

File: cpp/n292_NimGame.cpp
// https://leetcode-cn.com/problems/nim-game/submissions/
#include <iostream>
#include <vector>
#include <algorithm>
#include <string>
using namespace std;

class Solution {
public:
    bool canWinNim(int n) {
        return (n % 4 != 0);
    }
};

int main(){
    Solution solu;
    cout << solu.canWinNim(25) << endl;
}